### PR TITLE
bug 1559217: add stp block-lists for stage

### PR DESCRIPTION
**Pull request by groovecoder**

### CODE REVIEW

File: stage.ini [hunk 1]
```diff
@@ -49,11 +49,43 @@ output=analytics-track-digest256
...
+# NOTE: This is the *Disconnect* Social category; not the same as STP below
 [tracking-protection-social]
 categories=Social|Disconnect
 output=social-track-digest256
 s3_key=tracking/social-track-digest256
 
+# These "social-tracking-protection" lists are for Firefox STP
+[social-tracking-protection]
+disconnect_url=https://raw.githubusercontent.com/mozilla-services/shavar-prod-lists/master/social-tracking-protection-blacklist.json
+categories=Facebook,Twitter
```
Comment (englehardt) [inline stage.ini]: This should be `categories=Facebook|Twitter`. The `,` delimiter is used to specify multiple independent filters (i.e., and AND operation). `|` is used for OR.

Comment (groovecoder) [reply stage.ini]: Ah, got it. Thanks!

**Comment by groovecoder** [comment]
Updated with correct category filter syntax.

**Comment by englehardt** [comment]
Looks correct:

```
------ social-tracking-protection ------
-->blocklist: https://raw.githubusercontent.com/mozilla-services/shavar-prod-lists/master/social-tracking-protection-blacklist.json)
 * filter ['Facebook', 'Twitter'] matched 26 domains
 * removing 0 rule(s) due to DNT exceptions
Tracking protection(social-tracking-protection): publishing 27 items; file size 884

------ social-tracking-protection-facebook ------
-->blocklist: https://raw.githubusercontent.com/mozilla-services/shavar-prod-lists/master/social-tracking-protection-blacklist.json)
 * filter ['Facebook'] matched 21 domains
 * removing 0 rule(s) due to DNT exceptions
Tracking protection(social-tracking-protection-facebook): publishing 22 items; file size 724

------ social-tracking-protection-twitter ------
-->blocklist: https://raw.githubusercontent.com/mozilla-services/shavar-prod-lists/master/social-tracking-protection-blacklist.json)
 * filter ['Twitter'] matched 5 domains
 * removing 0 rule(s) due to DNT exceptions
Tracking protection(social-tracking-protection-twitter): publishing 6 items; file size 212

------ social-tracking-protection-linkedin ------
-->blocklist: https://raw.githubusercontent.com/mozilla-services/shavar-prod-lists/master/social-tracking-protection-blacklist.json)
 * filter ['LinkedIn'] matched 2 domains
 * removing 0 rule(s) due to DNT exceptions
Tracking protection(social-tracking-protection-linkedin): publishing 3 items; file size 115

------ social-tracking-protection-youtube ------
-->blocklist: https://raw.githubusercontent.com/mozilla-services/shavar-prod-lists/master/social-tracking-protection-blacklist.json)
 * filter ['YouTube'] matched 1 domains
 * removing 0 rule(s) due to DNT exceptions
Tracking protection(social-tracking-protection-youtube): publishing 2 items; file size 83
```

**Pull request merged**